KsApi/models/graphql/adapters/DeletePaymentSourceEnvelope+DeletePaymentSourceMutation.DataTests.swift

```diff
@@ -0,0 +1,81 @@
+@testable import KsApi
+import XCTest
+
+final class DeletePaymentSourceEnvelope_PaymentSourceDeleteMutationTests: XCTestCase {
+  func testPaymentSource_WithValidData_Success() {
+    // NOTE: Cannot convert directly from a String to the payment type, state or type. Apollo seems to return the Mutation.Data with these types created.
```

Comment:
Saw this note in the CreatePaymentSource tests too. What's the situation here? Is the credit card type being interpreted differently by Apollo?

Reply:
Ok, I removed these comments, they aren't that informational, its an indication that we can't directly pass strings like "visa" into a `[String:Any]` dictionary and have `GraphAPI` convert them to `Mutation.Data`, we need to pass in the `GraphAPI.CreditCardTypes.visa`. Just can't directly map a String to this type with `ResultMap`, getting some strange errors.